tap_googleads/client.py

```diff
@@ -21,7 +22,7 @@ def __init__(self, message: str, response: requests.Response) -> None:
 class GoogleAdsStream(RESTStream):
     """GoogleAds stream class."""
 
-    url_base = "https://googleads.googleapis.com/v18"
+    url_base = "https://googleads.googleapis.com/v20"
```

Comment:
I think we need to evaluate if there are any significant changes between these versions that we need to take into account:

https://developers.google.com/google-ads/api/diff-tool/v20/versus-v19/diff
https://developers.google.com/google-ads/api/diff-tool/v19/versus-v18/diff